### PR TITLE
fix load test for CRIB, update docs

### PR DESCRIPTION
**Pull request by skudasov**
Fix CRIB OCRv1 load test, update connect code and docs.
Should be merged after https://github.com/smartcontractkit/crib/pull/42

### CODE REVIEW

**Review by gheorghestrimtu — DISMISSED** [review]

**Review by gheorghestrimtu — DISMISSED** [review]

File: integration-tests/contracts/ethereum_contracts.go [hunk 1]
```diff
@@ -99,11 +99,6 @@ func DefaultOffChainAggregatorOptions() OffchainOptions {
 
 // DefaultOffChainAggregatorConfig returns some base defaults for configuring an OCR contract
 func DefaultOffChainAggregatorConfig(numberNodes int) OffChainAggregatorConfig {
-	if numberNodes <= 4 {
-		log.Err(fmt.Errorf("insufficient number of nodes (%d) supplied for OCR, need at least 5", numberNodes)).
-			Int("Number Chainlink Nodes", numberNodes).
-			Msg("You likely need more chainlink nodes to properly configure OCR, try 5 or more.")
-	}
```
Comment (Tofel) [inline integration-tests/contracts/ethereum_contracts.go:106]: why are we removing it? I see you check whether for CRIB we have at least 2 nodes, but why 2 if here we were expecting at least 5?

File: integration-tests/crib/connect.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+	if simulated {
+		netWSURL := fmt.Sprintf(ingressNetworkWSURLTemplate, cfg.Namespace, ingressSuffix)
+		netHTTPURL := fmt.Sprintf(ingressNetworkHTTPURLTemplate, cfg.Namespace, ingressSuffix)
+		setSethConfig(config, netWSURL, netHTTPURL)
+		net := blockchain.EVMNetwork{
+			Name:                 cfg.NetworkName,
+			Simulated:            true,
+			SupportsEIP1559:      true,
+			ClientImplementation: blockchain.EthereumClientImplementation,
+			ChainID:              1337,
```
Comment (Tofel) [inline integration-tests/crib/connect.go:72]: should we support 2337 chain as well? we launch CRIB with 2 chains, no?

Comment (skudasov) [reply integration-tests/crib/connect.go:72]: Not sure we need 2 chains for any tests in `core` right now

File: integration-tests/contracts/ethereum_contracts.go (hunk 1 above)
Comment (skudasov) [reply integration-tests/contracts/ethereum_contracts.go:106]: We count only worker nodes here, I can run it with 1+4

File: integration-tests/load/ocr/README.md [hunk 3]
```diff
@@ -1,28 +1,31 @@
...
-
-Create your cluster, if you already have one just use `kubefwd`
-```
-kubectl create ns cl-cluster
-devspace use namespace cl-cluster
-devspace deploy
-sudo kubefwd svc -n cl-cluster
+## Setup CRIB
+CRIB CORE user documentation is available in [CORE CRIB - Deploy & Access Instructions](https://smartcontract-it.atlassian.net/wiki/spaces/TT/pages/597197209/CORE+CRIB+-+Deploy+Access+Instructions)
+```toml
```
Comment (kalverra) [inline integration-tests/load/ocr/README.md]: Should be shell?

**Review by kalverra — CHANGES_REQUESTED** [review]

**Review by kalverra — APPROVED** [review]

**Review by Tofel — APPROVED** [review]

**Review by gheorghestrimtu — APPROVED** [review]

**Pull request merged**